ci/helpers/vpool.py

```diff
@@ -34,7 +34,7 @@ def get_vpools():
         Get all vpools on a cluster
 
         :return: vpools
-        :rtype: list
+        :rtype: list of ovs.dal.hybrids.vpool.VPool
```

Comment:
You need a return type not a sentence, a sentence can be specified in `return` or `returns`. 
This is because python IDE's use `rtype` as reference to look at when trying to validate a action on the used return type.